src/org/jgroups/util/SslContextFactory.java

```diff
@@ -121,15 +119,12 @@ public SslContextFactory trustStoreFileName(String trustStoreFileName) {
       return this;
    }
 
-   public SslContextFactory trustStorePassword(char[] trustStorePassword) {
-      this.trustStorePassword = trustStorePassword;
-      return this;
+   public SslContextFactory trustStorePassword(String trustStorePassword) {
+      return trustStorePassword(trustStorePassword.toCharArray());
```

Comment:
It needs a `null` check. 
If I don't configure a trust store, this method is invoked with `null`

Reply:
Fixed